integration-requirements.txt

```diff
@@ -1,3 +1,4 @@
+pytest-asyncio==0.21.1
```

Comment:
Pinning `pytest-operator==0.31.1` should do the trick.

Reply:
done